notebooks/7-rotation-speeds/interm_rotation_rate.ipynb

```diff
@@ -0,0 +1,2663 @@
+{
```

Comment:
<p>This could be a place for an exercise - they could view a zoomed-in version of the full light curve shown above to assess its shape. (Is it sinusoidal-ish?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>